app/views/maintenance.py

```diff
@@ -8,7 +8,12 @@
                          "SERVICE_UNAVAILABLE": "src/service-unavailable.html"}
 
 
+# Lots of options as a temporary fill in.
```

Comment:
This doesn't mean a whole lot.  Something like 'Hardcode the most common routes until we find a wildcard solution' (or something like that) would be better